libs/wingsdk/src/simulator/simulator.ts

```diff
@@ -412,6 +423,14 @@ export class Simulator {
     return treeData.display?.ui ?? [];
   }
 
+  private getResourceStateDir(resourcePath: string) {
+    const hash = createHash("sha256")
```

Comment:
For later - We should include the resource `addr` in the simulator.json so there won't be a need to calculate a hash every time.